### PR TITLE
DocumentDB: Fixed failoverPolicies request body (#701)

### PR DESCRIPTION
**Pull request by dmakwana**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X] If applicable, the PR references the bug/issue that it fixes.
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] Validation errors from the [Linter extension for VS Code](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/linter.md) have all been fixed for this spec. (**Note:** for large, previously checked in specs, there will likely be many errors shown. Please contact our team so we can set a timeframe for fixing these errors if your PR is not going to address them).
  - [X] The spec follows the patterns described in the [Swagger good patterns](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-good-patterns.md) document unless the service API makes this impossible.

---
This change is [Reviewable](https://reviewable.io/reviews/azure/azure-rest-api-specs/702)

### CODE REVIEW

File: arm-documentdb/2015-04-08/swagger/documentdb.json [hunk 1]
```diff
@@ -312,6 +312,7 @@
       "post": {
         "operationId": "DatabaseAccounts_RegenerateKey",
         "description": "Regenerates an access key for the specified Azure DocumentDB database account.",
+        "x-ms-long-running-operation": true,
```
Comment (amarzavery) [inline arm-documentdb/2015-04-08/swagger/documentdb.json]: IF regenerateKey is a long running operation then what is the terminal status code? I mean there should be a 200 documented as well, that would indicate AutoRest to deserialized the response based on some schema.

File: arm-documentdb/2015-04-08/swagger/documentdb.json [hunk 2]
```diff
@@ -380,11 +381,16 @@
       "description": "The List operation response, that contains the database accounts and their properties."
     },
     "FailoverPolicies": {
-      "type": "array",
-      "description": "List of failover policies.",
-      "items": {
-        "$ref": "#/definitions/FailoverPolicy"
-      }
+      "properties": {
+        "failoverPolicies": {
```
Comment (amarzavery) [inline arm-documentdb/2015-04-08/swagger/documentdb.json]: This looks like a breaking change.

Comment (lmazuel) [inline arm-documentdb/2015-04-08/swagger/documentdb.json]: @amarzavery was not working before. It's a working change :)
https://github.com/Azure/azure-rest-api-specs/issues/701

File: arm-documentdb/2015-04-08/swagger/documentdb.json (hunk 1 above)
Comment (dmakwana) [inline arm-documentdb/2015-04-08/swagger/documentdb.json]: This is the same case as with the DatabaseAccounts_CreateOrUpdate where we have the location value in the response header

Comment (johanste) [inline arm-documentdb/2015-04-08/swagger/documentdb.json]: We still need the eventual return value documented - just as DatabaseAccounts_CreateOrUpdate documents the 200 response. 

---
In reply to: [87484179](https://github.com/Azure/azure-rest-api-specs/pull/702#discussion_r87484179) [](ancestors = 87484179)

Comment (dmakwana) [inline arm-documentdb/2015-04-08/swagger/documentdb.json]: Addressed

**Pull request merged**